### PR TITLE
Resets map style and button selection to 'pro' when setupQuiz() is called

### PR DESCRIPTION
**Pull request by brendon1555**
Implementation of #9

### CODE REVIEW

File: app/assets/js/main.js [hunk 1]
```diff
@@ -193,6 +200,8 @@
 		_app.quiz.answer = _app.quiz.answer || '';
 		_app.quiz.region = _app.quiz.region || 'World';
 
+		resetStyle();
```
Comment (jabranr) [inline app/assets/js/main.js]: I would rather rename it so it reads more relative to UI actions i.e. `resetLevel()`?

File: app/assets/js/main.js [hunk 2]
```diff
@@ -185,6 +185,13 @@
 		}));
 	}
 
+	// Reset to pro (map style & button selection)
+	function resetStyle() {
+		setStyle(_app.map, 'pro');
```
Comment (jabranr) [inline app/assets/js/main.js]: Here would you consider reusing the already attached event listener at line 85? 

Such as:

```
$('.mode-btn[data-level="pro"]').click();
```

File: app/assets/js/main.js (hunk 1 above)
Comment (brendon1555) [reply app/assets/js/main.js]: I'm on it

File: app/assets/js/main.js (hunk 2 above)
Comment (brendon1555) [reply app/assets/js/main.js]: ah yep good point

File: app/assets/js/main.js [hunk 3]
```diff
@@ -186,10 +186,9 @@
 	}
 
 	// Reset to pro (map style & button selection)
-	function resetStyle() {
+	function resetLevel() {
 		setStyle(_app.map, 'pro');
```
Comment (jabranr) [inline app/assets/js/main.js]: Thanks for the changes. 

I think you do not need this `setStyle` function call as it is already part of the `click` event handler. It is only duplicating it.

Comment (brendon1555) [reply app/assets/js/main.js]: good point! I'll clean that up.

**Comment by jabranr** [comment]
@brendon1555 thank you for your contribution 🎉

**Pull request merged**